models/DataObject/ClassDefinition/Data/Block.php

```diff
@@ -173,10 +170,8 @@ public function getDataFromResource($data, $object = null, $params = [])
 
             foreach ($unserializedData as $blockElements) {
                 $items = [];
-                /** @var $blockElement DataObject\Data\BlockElement */
+                /** @var DataObject\Data\BlockElement $blockElement */
```

Comment:
$blockElementRaw

Reply:
$blockElementDataRaw should be a array. $blockElement is a few lines below, we can remove this.